widgets/Plus/P.js

```diff
@@ -25,7 +25,9 @@ class P extends React.Component{
     componentDidMount () {
        WindowActions.loginMessage.listen((jsmsg) => {
             if (jsmsg.profile) {
-                this.userId(jsmsg.profile.id);
+                Actions.plusActive(false, 'https://wr.io/' + jsmsg.profile.id + '/Plus-WRIO-App/', () => {
```

Comment:
@AlexeyAnshakov Seems to be ok for now. For future better to extract this domain to some variable, so it can be changed easily.